pkg/document/webresolver/resolvehandler.go

```diff
@@ -64,8 +72,19 @@ func (r *ResolveHandler) ResolveDocument(id string) (*document.ResolutionResult,
 		r.metrics.WebDocumentResolveTime(time.Since(startTime))
 	}()
 
+	hostWithPort, err := util.BetweenStrings(id, "did:web:", ":scid:")
```

Comment:
Not sure if the Parse DID function would be better to use here: https://github.com/hyperledger/aries-framework-go/blob/f7a0dc3a00bd4c8139e8c89e25d027e9093928a9/pkg/doc/did/doc.go#L92

Reply:
It would be better validation - I will change it.

Reply:
Fixed